### PR TITLE
update eslint no use import

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- update eslint `unuse import`:  error in production , warning in dev local.    (tested, see `actions` tab)
+in CI: image
+local: 
image

- update minor ux type && swap:  press enter import token will make list pair showing

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  extends: [
+    'react-app',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:prettier/recommended',
+  ],
+  plugins: ['better-styled-components', 'unused-imports'],
+  rules: {
+    'unused-imports/no-unused-imports': process.env.NODE_ENV === 'production' ? 'error' : 'warn',
```
Comment (nguyenhoaidanh) [inline .eslintrc.js:44]: only update 1 line.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    ecmaFeatures: {
+      // Allows for the parsing of JSX
+      jsx: true,
+    },
+  },
+  ignorePatterns: [
+    'src/locales/**/*.js',
+    'src/locales/**/en-US.po',
```
Comment (namgold) [inline .eslintrc.js:13]: anyone have idea why this is not `*.po`? @frontend-engineers

Comment (hungdoansy) [reply .eslintrc.js:13]: We can remove these `po` files, eslint won't check them

File: .eslintrc.js (hunk 1 above)
Comment (hungdoansy) [reply .eslintrc.js:44]: Nice

File: src/pages/Pools/FarmingPoolsMarquee.tsx [hunk 3]
```diff
@@ -132,16 +132,6 @@ const FarmingPoolsMarquee = ({ tab }: { tab: string }) => {
 
 export default FarmingPoolsMarquee
 
-const IconWrapper = styled.div`
```
Comment (hungdoansy) [inline src/pages/Pools/FarmingPoolsMarquee.tsx:135]: Is this included? 🤔

File: src/components/swapv2/PairSuggestion/index.tsx [hunk 4]
```diff
@@ -221,6 +221,7 @@ export default forwardRef<PairSuggestionHandle, Props>(function PairSuggestionIn
         search: stringify(newQs),
       })
       setShowModalImportToken(true)
+      refInput.current?.blur()
```
Comment (hungdoansy) [inline src/components/swapv2/PairSuggestion/index.tsx:224]: Is this included? 🤔

File: src/pages/Pools/FarmingPoolsMarquee.tsx (hunk 3 above)
Comment (nguyenhoaidanh) [reply src/pages/Pools/FarmingPoolsMarquee.tsx:135]: just remove unuse var.  not related main purpose of this PR :D

File: src/components/swapv2/PairSuggestion/index.tsx (hunk 4 above)
Comment (nguyenhoaidanh) [reply src/components/swapv2/PairSuggestion/index.tsx:224]: I added description.   
image

**Review by hungdoansy — APPROVED** [review]
LGTM

**Pull request merged**